appveyor.yml

```diff
@@ -7,10 +7,13 @@ environment:
 
 services:
   - mssql2016
+  
+cache:
+  - C:\Users\appveyor\.m2 -> pom.xml
```

Comment:
changing to $HOME didnt work. 

With that update, appveryor can't find the cache, downloads all maven dependencies, and then creates a new build cache on every build.

Reverting back to original implementation